internal/providers/terraform/parser.go

```diff
@@ -327,6 +332,28 @@ func parseTags(resourceType string, v gjson.Result) map[string]string {
 	}
 }
 
+func overrideRegion(addr string, resourceType string, config *config.Config) string {
+	region := ""
+	providerPrefix := strings.Split(resourceType, "_")[0]
```

Comment:
```suggestion
	providerPrefix := strings.Split(resourceType, "_")
	if len(providerPrefix) == 0 {
	    return ""
	}
```

so we don't panic with invalid provider strings (vscode)

Reply:
Going to do the same in other methods 👍